lib/libteleirc.js

```diff
@@ -433,6 +517,19 @@ class TeleIrc {
             return firstName + " (@" + userName + ") " + suffix;
         }
     }
+
+    /**
+     * Sends the given Telegram message from a USER to the IRC channel.
+     * This includes the IRC prefix and the suffix.
+     * @param {String} from - Who the message was from on the Telegram Side. 
+     * @param {String} message - The message to send to the IRC channel.
+     */
+    sendUserMessageToIrc(from, message) {
```

Comment:
I was wondering where this method came from, didn't see it until I worked my way to the bottom. Nice idea, this is a little more clean. 👍